engines/onboarding/app/controllers/onboarding/api/v1/invitations_controller.rb

```diff
@@ -2,10 +2,20 @@ module Onboarding
   module Api
     module V1
       class InvitationsController < ApplicationController
-        before_action :authenticate, only: [:bulk]
-        before_action :load_group, only: [:bulk]
+        before_action :authenticate, only: [:index, :bulk]
+        before_action :load_group, only: [:index, :bulk]
         before_action :load_invitation, only: [:show, :accept]
 
+        # GET /api/v1/invitations?group_id=:id
+        #
+        def index
+          authorize @group
+
+          invitations = Invitation.where(group_id: @group.id)
```

Comment:
So you can get all invitations of all your groups. Right?

For me (Katuma UI) is not important. But maybe in the future (Do not change now please) is interesting as an API consumer